### PR TITLE
New package: actualsolution.VolumeLock version 2.4

### PR DESCRIPTION
**Pull request by soredake**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151305)

### CODE REVIEW

**Comment by stephengillie** [comment]
Not seeing a cause for the [Policy 2.7](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#27-adult-content) check.

File: manifests/a/actualsolution/VolumeLock/2.4/actualsolution.VolumeLock.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: actualsolution.VolumeLock
+PackageVersion: '2.4'
```
Comment (stephengillie) [inline manifests/a/actualsolution/VolumeLock/2.4/actualsolution.VolumeLock.installer.yaml:5]:
```suggestion
PackageVersion: 2.4.1
```
Please update file path to match this change.

File: manifests/a/actualsolution/VolumeLock/2.4/actualsolution.VolumeLock.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: actualsolution.VolumeLock
+PackageVersion: '2.4'
```
Comment (stephengillie) [inline manifests/a/actualsolution/VolumeLock/2.4/actualsolution.VolumeLock.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.4.1
```
Please update file path to match this change.

File: manifests/a/actualsolution/VolumeLock/2.4/actualsolution.VolumeLock.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: actualsolution.VolumeLock
+PackageVersion: '2.4'
```
Comment (stephengillie) [inline manifests/a/actualsolution/VolumeLock/2.4/actualsolution.VolumeLock.yaml:5]:
```suggestion
PackageVersion: 2.4.1
```
Please update file path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @soredake,

If possible, I believe the `PackageVersion` should match the Registry. Otherwise, this PR appears ready to merge.

**Comment by soredake** [comment]
Done.

**Comment by soredake** [comment]
I don't see any errors on local validation.

**Comment by stephengillie** [comment]
It's an error that doesn't show up in manual validation. 
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\a\actualsolution\VolumeLock\2.4', 
expected 'manifests\a\actualsolution\VolumeLock\2.4.1'

**Comment by soredake** [comment]
Changed version to 2.4.1

**Review by stephengillie — APPROVED** [review]

**Pull request merged**